### PR TITLE
variables renaming 

### PR DESCRIPTION
**Pull request by adesokanayo**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -15,64 +15,64 @@ import (
...
 func main() {
 	// parse arguments
-	grpc_port := flag.Int("grpc-port", 5005, "port number for gRPC server to listen on")
+	grpcPort := flag.Int("grpcPort", 5005, "port number for gRPC server to listen on")
 	capacity := flag.Int("capacity", 1000, "capacity of LRU cache")
-	client_auth := flag.Bool("-enable-client-auth", false, "require client authentication (used for mTLS)")
-	https_enabled := flag.Bool("enable-https", false, "enable HTTPS for server-server and client-server communication. Requires TLS certificates in /certs directory.")
-	config_file := flag.String("config", "", "filename of JSON config file with the info for initial nodes")
-	rest_port := flag.Int("rest-port", 8080, "enable REST API for client requests, instead of just gRPC")
+	clientAuth := flag.Bool("clientAuth", true, "require client authentication (used for mTLS)")
```
Comment (malwaredllc) [inline main.go:20]: Can you change this to `enableHttps` and `enableClientAuth`

File: Dockerfile.client [hunk 2]
```diff
@@ -9,7 +9,7 @@ RUN go mod download && go mod verify
 
 COPY . .
 
-WORKDIR /usr/src/app/client/cache_client
```
Comment (malwaredllc) [inline Dockerfile.client:12]: Can you change the package name to `client`? I want to follow the go package naming conventions: https://go.dev/blog/package-names

Note that you will also need move the .go files out of `cache_client/` up a directory and then delete the `cache_client` directory

**Comment by malwaredllc** [comment]
Thanks for your work on this @adesokanayo - I added a few notes for small changes that need to be addressed, and I also ran the tests, which failed unfortunately. However, please note you can also run any of these tests locally. You can see which tests run in the CI by looking [here](https://github.com/malwaredllc/minicache/blob/main/.github/workflows/go.yml#L29-L42)

File: docker-compose.yml [hunk 3]
```diff
@@ -47,6 +47,6 @@ services:
       dockerfile: Dockerfile.server
     restart: unless-stopped
     environment: # Pass environment variables to the service
-      GIN_MODE: release
+      GINMODE: release
```
[the remark this answers is not in the document]
Comment (adesokanayo) [reply docker-compose.yml:50]: Done

**Comment by adesokanayo** [comment]
@malwaredllc  I have only the integration test failing.  Do you think there is a config  mismatch in my changes ?

Screen Shot 2022-05-23 at 9 08 24 AM

**Comment by malwaredllc** [comment]
@adesokanayo It looks like it's because you need to update the Node struct JSON serialization/deserialization [here](https://github.com/malwaredllc/minicache/blob/main/node/node.go#L33-L34). You change the key names in the config file so you need to update that.

**Comment by adesokanayo** [comment]
If the workflow still fails, I will start from a fresh clone of main  and will start with function-scoped variables only.

**Comment by malwaredllc** [comment]
@adesokanayo Tests are passing but can you change the name of `cacheClient.go` to `client.go`, and `cacheClient_Docker_Test.go` to `client_test.go`?

**Comment by malwaredllc** [comment]
@adesokanayo Nice, I am glad you changed the `lru_cache` to `lru` as well. You will need to update the CI test here to use that new name: https://github.com/malwaredllc/minicache/blob/main/.github/workflows/go.yml

**Comment by malwaredllc** [comment]
Merged! Thanks for your work on this @adesokanayo

**Pull request merged**